.eslintrc.js

```diff
@@ -13,5 +13,22 @@ module.exports = {
                 lintFile: './tslint.json', // path to tslint.json of your project
             },
         ],
+        'dcr/only-import-below': [
+            'warn',            
+            {
+                allowedImports: [
+                    "react",
+                    "emotion",
+                    "jsdom",
+                    "curlyquotes",
+                    "react-dom",
+                    "@guardian/pasteup",
+                    "@frontend/lib/",
+                    "@frontend/amp/lib/",
+                    "@testing-library",
+                    "@root/packages/frontend/amp/lib/"
```

Comment:
Presumably the plan is for us to refactor any uses of this separately and then remove this line?

Reply:
I'd just added in the parts of the codebase that seemed to have heavy reuse right now. 

I'm not sure what we actually want to make the official list of allowed imports. There's still going to be a dozen or so warnings after this lint rule goes in regardless!